docs/README.md

```diff
@@ -0,0 +1,37 @@
+## The Documentation Website
+
+This folder holds the source code of our documentation website:
+[kotlin.github.io/dataframe](https://kotlin.github.io/dataframe).
+
+It's built using [WriterSide](https://www.jetbrains.com/writerside/) and published
+by a [Github Action](../.github/workflows/main.yml).
+The file structure mirrors the default WriterSide structure with some small exceptions.
+
+Images all README files can be stored in [docs/imgs](./imgs).
+
+### Explainer dataframes
+`@TransformDataFrameExpressions` annotated test functions generate sample
+dataframe HTML files that can be used as iFrames on the documentation website.
+They are tested, generated, and copied over to [docs/StardustDocs/snippets](./StardustDocs/snippets) by
+our "explainer" [plugin callback proxy](../core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/explainer),
+which hooks into [the TestBase class](../core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/TestBase.kt) and
+retrieves the intermediate DataFrame expressions thanks to
+our "explainer" compiler plugin [:plugins:expressions-converter](../plugins/expressions-converter).
+
+We can also generate "normal" DataFrame samples for the website. This can be done using the
+[OtherSamples class](../core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/OtherSamples.kt). Generated
+HTML files will be stored in [docs/StardustDocs/snippets/manual](./StardustDocs/snippets/manual).
+
+### KDoc Preprocesor
+KDocs can also be exported to HTML, for them to be reused on the website.
```

Comment:
Is it copypaste from core/Readme?

Reply:
yes, well, only the part relevant to the docs